vendor/modules.txt

```diff
@@ -2242,7 +2242,7 @@ k8s.io/controller-manager/pkg/informerfactory
 k8s.io/controller-manager/pkg/leadermigration
 k8s.io/controller-manager/pkg/leadermigration/config
 k8s.io/controller-manager/pkg/leadermigration/options
-# k8s.io/cri-api v0.20.1 => github.com/openshift/kubernetes/staging/src/k8s.io/cri-api v0.0.0-20221020223141-4bd0702b5791
+# k8s.io/cri-api v0.0.0 => github.com/openshift/kubernetes/staging/src/k8s.io/cri-api v0.0.0-20221020223141-4bd0702b5791
```

Comment:
This change is weird. It's the same SHA, but the version changed. Does that version matter?

Reply:
I don't think so, all k8s.io/* are replaced to v0.0.0-*
What's interesting, this rebase is based on the same image as my manual rebase (https://github.com/openshift/microshift/pull/1055) and yet there are changes... Probably it's coming from different ARM image